seqr/views/apis/anvil_workspace_api_tests.py

```diff
@@ -451,16 +513,16 @@ def test_get_anvil_vcf_list(self, mock_subprocess, mock_file_logger, mock_utils_
 
         # Test empty bucket
         mock_subprocess.return_value.wait.return_value = 0
-        mock_subprocess.return_value.stdout = b''
+        mock_subprocess.return_value.stdout = []
         response = self.client.get(url, content_type='application/json')
         self.assertEqual(response.status_code, 200)
         self.assertDictEqual(response.json(), {'dataPathList': []})
-        mock_subprocess.assert_has_calls([
```

Comment:
why did this change?

Reply:
I don't want to test the `wait()` called and make the test cleaner.